src/services/tweets.sevice.ts

```diff
@@ -0,0 +1,63 @@
+import { ObjectId } from "mongodb";
+
+import {
+  idValidator,
+  stringValidator,
+} from "../helpers/tweets.validators.utils";
+import {
+  createOne,
+  deleteOne,
+  getAll,
+  getOneById,
+  updateOne,
+} from "../repositories/tweet.repository";
+
+export const getTweetById = (id: string) => {
+  idValidator(id);
+  return getOneById(new ObjectId(id));
+};
+
+export const createTweet = async (
+  authorId: string,
+  content: string,
+  repliedTo?: string
+) => {
+  idValidator(authorId);
+  idValidator(repliedTo);
+  stringValidator(content, "content");
+  createOne(new ObjectId(authorId), content, new ObjectId(repliedTo));
+};
+export const deleteTweet = async (id: string) => {
+  idValidator(id);
+  return deleteOne(new ObjectId(id));
```

Comment:
send string parameters and in the repository get new ObjectId(id)
idValidator move to repository too